SECURITY.md

```diff
@@ -0,0 +1,9 @@
+# Security Policy
+
+## Supported Versions
+
+See our [Version Compatibility Documentation](https://docs.konghq.com/kubernetes-ingress-controller/1.1.x/references/version-compatibility/#kubernetes).
+
+## Reporting a Vulnerability
+
+See our [Reporting A Vulnerability Documentation](https://docs.konghq.com/enterprise/2.4.x/kong-security-update-process/#reporting-a-vulnerability).
```

Comment:
Link should be to /latest/ instead of /2.4.x/ or /1.1.x/.  There's a redirect in place to help keep that up to date.  Other than that, LGTM

Reply:
```suggestion
See our [Reporting A Vulnerability Documentation](https://docs.konghq.com/enterprise/latest/kong-security-update-process/#reporting-a-vulnerability).
```